src/script/page/LeftSidebar/panels/StartUI/StartUI.tsx

```diff
@@ -100,28 +100,26 @@ const StartUI: React.FC<StartUIProps> = ({
 
   const peopleSearchResults = useRef<SearchResultsData | undefined>(undefined);
 
-  const openFirstConversation = (): void => {
+  const openFirstConversation = async (): Promise<void> => {
```

Comment:
you actually don't need to mark this one async. 

The rule of thumb is: you only need to put `async` if you need `await` in the body of the function. 

Else a simple `: Promise<...>` signature is enough :) 

```suggestion
  const openFirstConversation = (): Promise<void> => {
```

Reply:
I think I'm missing something obvious :thinking: 
![image](https://user-images.githubusercontent.com/78490891/215499012-84a31f40-36ca-4496-94b3-a5709e4e9ae9.png)

Reply:
I guess you can just remove the return type at all, it's just extra maintenance here :) 
TypeScript is going to infer it for you ;) 

```suggestion
  const openFirstConversation = () => {
```